main.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+# Module: plugin.video.magaritv
+# Version: 1.0.0
+# Author: Orazio Giliberti
+# Created on: 29.04.2016
+# License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+# This module was a modificated version of
+# https://github.com/romanvm/plugin.video.example/releases
+# released by Roman V M under License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+# Thanks Roman !
+
+
+import sys
+import json
+from urlparse import parse_qsl
+import xbmcgui
+import xbmcplugin
+
+
+__url__ = sys.argv[0]
```

Comment:
Next update please remove \__varname\__ style labels. These should only be used for python internals.